### PR TITLE
DDF-2607 upgrade to artemis 1.5.1

### PR DESCRIPTION
**Pull request by ryeats**
#### What does this PR do?
Upgrades artemis to 1.5.1 to get some bug fixes we need
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@jrnorth 
@harrison-tarr 
@roelens8  
@dcruver 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Build](https://github.com/orgs/codice/teams/build)
[Docs](https://github.com/orgs/codice/teams/docs)
[UI](https://github.com/orgs/codice/teams/ui)
[Security](https://github.com/orgs/codice/teams/security)
[Continuous Integration](https://github.com/orgs/codice/teams/continuous-integration)
[Solr](https://github.com/orgs/codice/teams/solr)
[IO](https://github.com/orgs/codice/teams/IO)
[OGC](https://github.com/orgs/codice/teams/OGC)
[Data](https://github.com/orgs/codice/teams/data)
[Core APIs](https://github.com/orgs/codice/teams/core-apis)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jaymcnallie
@kcwire
#### How should this be tested? (List steps with links to updated documentation)
feature:install artemis-amqp
then verify that the artemis starts up in jconsole and it has the correct acceptor
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: platform/platform-app/src/main/resources/features.xml [hunk 1]
```diff
@@ -1046,15 +1046,25 @@
...
-        <feature prerequisite="true">scr</feature>
+        <feature>transaction</feature>
+        <feature>netty-core</feature>
+        <feature>scr</feature>
         <configfile finalname="etc/org.apache.activemq.artemis.cfg">mvn:org.apache.activemq/artemis-features/${artemis.version}/cfg</configfile>
         <configfile finalname="etc/artemis.xml">mvn:org.apache.activemq/artemis-features/${artemis.version}/xml/artemis</configfile>
 
         <bundle>mvn:org.apache.geronimo.specs/geronimo-jms_2.0_spec/${geronimo-jms_2_spec.version}</bundle>
-        <bundle dependency="true">mvn:com.google.guava/guava/${guava.version}</bundle>
+        <bundle>mvn:com.google.guava/guava/19.0</bundle>
```
Comment (harrison-tarr) [inline platform/platform-app/src/main/resources/features.xml]: why not use `{$guava.version}`? is it an older version?

Comment (ryeats) [inline platform/platform-app/src/main/resources/features.xml]: yes and things didn't work with the newer version if I tried to upgrade ddf's guava, but not in obvious fixable ways or I would have just fixed them.

File: distribution/ddf-common/src/main/resources/etc/artemis.xml [hunk 2]
```diff
@@ -36,9 +36,28 @@
...
+        <max-disk-usage>90</max-disk-usage>
+
+        <!-- the system will enter into page mode once you hit this limit.
+             This is an estimate in bytes of how much the messages are using in memory -->
+        <global-max-size>104857600</global-max-size>
+
         <acceptors>
             <!-- AMQP Acceptor.  Listens on default AMQP port for AMQP traffic.-->
-            <acceptor name="amqp">tcp://0.0.0.0:${artemis.amqp.port}?sslEnabled=true;keyStorePath=${javax.net.ssl.keyStore};keyStorePassword=${javax.net.ssl.keyStorePassword};enabledCipherSuites=${https.cipherSuites};enabledProtocols=${https.protocols}</acceptor>
+            <!--<acceptor name="amqp">tcp://0.0.0.0:${artemis.amqp.port}?sslEnabled=true;keyStorePath=${javax.net.ssl.keyStore};keyStorePassword=${javax.net.ssl.keyStorePassword};enabledCipherSuites=${https.cipherSuites};enabledProtocols=${https.protocols}</acceptor>-->
```
Comment (jrnorth) [inline distribution/ddf-common/src/main/resources/etc/artemis.xml]: Remove

File: distribution/ddf-common/src/main/resources/etc/artemis.xml [hunk 3]
```diff
@@ -36,9 +36,28 @@
...
+
+            <!-- Default ActiveMQ Artemis Acceptor.  Multi-protocol adapter.  Currently supports Core, OpenWire, Stomp and AMQP. -->
+            <!-- performance tests have shown that openWire performs best with these buffer sizes -->
```
Comment (jrnorth) [inline distribution/ddf-common/src/main/resources/etc/artemis.xml]: Should we be setting the buffer sizes? If not, this comment should be removed.

Comment (ryeats) [inline distribution/ddf-common/src/main/resources/etc/artemis.xml]: ah yeah since we don't have openwire by default ill remove it

**Pull request merged**